src/theory/inference_id.h

```diff
@@ -229,6 +236,12 @@ enum class InferenceId
   DATATYPES_SYGUS_MT_POS,
   // ---------------------------------- end datatypes theory
 
+  //-------------------------------------- floating point theory
+  FP_PREPROCESS,
+  FP_EQUATE_TERM,
+  FP_REGISTER_TERM,
```

Comment:
Some docs here as well? Should we split the FP_PREPROCESS into more specific ids?

Reply:
I'd prefer this level of granularity for now, as I'm not sure if the ways that FP sends lemmas will be further refactored.  @aniemetz do you have plans to change how FP makes calls to lemma?